qabelbox/src/main/java/de/qabel/qabelbox/helper/Formatter.kt

```diff
@@ -0,0 +1,43 @@
+package de.qabel.qabelbox.helper
+
+import android.content.Context
+import android.content.res.Resources
+import android.text.format.DateUtils
+
+import java.text.DateFormat
+import java.text.DecimalFormat
+import java.util.Date
+import java.util.regex.Pattern
+
+import de.qabel.qabelbox.R
+
+object Formatter {
+
+    private val dateTimeFormat by lazy { DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT) }
+    private val dateFormat by lazy { DateFormat.getDateInstance(DateFormat.MEDIUM) }
+    private val EMAIL_PATTERN by lazy { Pattern.compile("^.+@.+$") }
+
+    fun formatDateShort(date: Date): String =
+            dateFormat.format(date)
+
+    fun formatDateTimeShort(date: Date): String =
+            dateTimeFormat.format(date)
+
+    /**
+     * Formats a Date[time] to a String like "moments ago" if [time] is less than a mintes in the past or
+     * "Fr. 20:35" or "15.02.15 20:35"
+     * if the [time] is more than a week in the past.
+
+     * @param time
+     * @return
```

Comment:
as you touched the docblock: remove the params as they are wrong and one can guess what the input and output means by reading that comment